src/Service/Resolvers/Sql Query Structures/SqlQueryStructure.cs

```diff
@@ -149,16 +149,7 @@ public class SqlQueryStructure : BaseSqlQueryStructure
             foreach (KeyValuePair<string, object> predicate in context.PrimaryKeyValuePairs)
             {
                 sqlMetadataProvider.TryGetBackingColumn(EntityName, predicate.Key, out string? backingColumn);
-                PopulateParamsAndPredicates(field: predicate.Key,
-                                            backingColumn: backingColumn!,
-                                            value: predicate.Value);
-            }
-
-            foreach (KeyValuePair<string, object?> predicate in context.FieldValuePairsInBody)
-            {
-                sqlMetadataProvider.TryGetBackingColumn(EntityName, predicate.Key, out string? backingColumn);
```

Comment:
How did you verify that this is unused code?

Reply:
When constructing the `FindRequestContext`, the data received from the request body is simply ignored. But, for `InsertRequestContext`, `UpsertRequestContext` there is logic to populate `FieldValuePairsInBody`. 

I manually verified this by setting debug points in `SqlQueryStructure` class anc examining the contents of the `context`. I tried this for requests - with and without a request body. In all cases, the `context.PopulateFieldValuePairsInBody` is empty.

I've added tests for GET requests with non-empty request body.